package.json

```diff
@@ -42,8 +42,8 @@
   "prepublish": "npm run build"
  },
  "dependencies": {
-  "vis": "4.21.0",
-  "@types/vis": "4.21.19"
+  "vis-network": "5.4.1",
+  "vis-timeline": "5.1.0"
```

Comment:
The types for Timeline has been merged but not released yet. Maybe we should wait for the next version?

Reply:
yes i would like to wait for that